x-pack/plugins/cloud_defend/server/lib/telemetry/collectors/schema.ts

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { MakeSchemaFrom } from '@kbn/usage-collection-plugin/server';
+import type { CloudDefendUsage } from './types';
+
+export const cloudDefendUsageSchema: MakeSchemaFrom<CloudDefendUsage> = {
+  indices: {
+    alerts: {
+      doc_count: {
+        type: 'long',
+      },
+      deleted: {
+        type: 'long',
+      },
+      size_in_bytes: {
+        type: 'long',
+      },
+      last_doc_timestamp: {
+        type: 'date',
+      },
+    },
+    file: {
+      doc_count: {
+        type: 'long',
+      },
+      deleted: {
+        type: 'long',
+      },
+      size_in_bytes: {
+        type: 'long',
+      },
+      last_doc_timestamp: {
+        type: 'date',
+      },
+    },
+    process: {
+      doc_count: {
+        type: 'long',
+      },
+      deleted: {
+        type: 'long',
+      },
+      size_in_bytes: {
+        type: 'long',
+      },
+      last_doc_timestamp: {
+        type: 'date',
+      },
+    },
+    latestPackageVersion: { type: 'keyword' },
+    packageStatus: {
+      status: { type: 'keyword' },
+      installedPackagePolicies: { type: 'long' },
+      healthyAgents: { type: 'long' },
+    },
+  },
+  pods_stats: {
+    type: 'array',
+    items: {
+      account_id: { type: 'keyword' },
+      container_image_name: { type: 'keyword' },
+      container_image_tag: { type: 'keyword' },
```

Comment:
It might be nice to see pod_namespace in here in the future. I could see us adding rules to ignore `kube-system` if it becomes all-consuming and we have trouble getting the signal from the noise.

Reply:
hmm, ok next time, plz "request changes" instead of "approving" i had auto merge on.

Reply:
ah, sorry didn't see the "in the future" bit. Namespace would be helpful yes. I can open another PR if you want it for this release.

Reply:
I should be the one apologizing. It was an afterthought for me. I will bring it up with @nick-alayil  when he returns, and we can cut a ticket to add it.